src/handler.ts

```diff
@@ -56,6 +56,10 @@ export abstract class ConnectionHandler {
       .onReady()
       .then(
         () => {
+          this.languageClient.onTelemetry(event => {
+            const eventName = event.Name ? event.Name : 'PUPPET_LANGUAGESERVER_EVENT';
+            reporter.sendTelemetryEvent(eventName, event.Measurements, event.Properties);
```

Comment:
Where are the `Measurements` and `Properties` methods coming from?  I can't see them in the Language Client node module.  And the data coming from the server is `any` so there are no guarantees here.